dev/create-release/release-build.sh

```diff
@@ -219,7 +219,7 @@ if [[ "$1" == "package" ]]; then
 
     # Write out the VERSION to PySpark version info we rewrite the - into a . and SNAPSHOT
     # to dev0 to be closer to PEP440.
-    PYSPARK_VERSION=`echo "$SPARK_VERSION" |  sed -r "s/-/./" | sed -r "s/SNAPSHOT/dev0/"`
+    PYSPARK_VERSION=`echo "$SPARK_VERSION" |  sed -r "s/-/./" | sed -r "s/SNAPSHOT/dev0/" | sed -r "s/preview/dev0/"`
```

Comment:
Oops. @jiangxb1987 .
This is used here.
```
PYTHON_DIST_NAME=pyspark-$PYSPARK_VERSION.tar.gz
```

Reply:
IIUC it would be `pyspark-3.0.0.dev0.tar.gz` which is expected ?

Reply:
Oh, I misread the code. You're right.

Reply:
At least for `tarball` name?

Reply:
super nit: How about using `-e` instead of using multiple sed commands if we have a change to update further.
https://unix.stackexchange.com/questions/33157/what-is-the-purpose-of-e-in-sed-command

Reply:
I tested locally and it generates:
```
-rw-r--r--  1 ubuntu ubuntu 218128015 Oct 30 05:16 pyspark-3.0.0.dev0.tar.gz
-rw-r--r--  1 ubuntu ubuntu       833 Oct 30 05:16 pyspark-3.0.0.dev0.tar.gz.asc
-rw-r--r--  1 ubuntu ubuntu       225 Oct 30 05:16 pyspark-3.0.0.dev0.tar.gz.sha512
```

Reply:
So, are you going to rename them manually before uploading for Apache Spark VOTE, @jiangxb1987 ? Or, do you want to use that `pyspark-3.0.0.dev0.tar.gz` directly?